src/simplicial-hott/04a-jorthogonality.rzk.md

```diff
@@ -0,0 +1,126 @@
+# 4a. Right orthogonal fibrations
+
+This is a literate `rzk` file:
+
+```rzk
+#lang rzk-1
+```
+
+For every shape inclusion `ϕ ⊂ ψ`,
+we obtain a fibrancy condition for a map `α : A' → A`
+in terms of unique extension along `ϕ ⊂ ψ`.
+This is a relative version of unique extension along `ϕ ⊂ ψ`.
+
+We say that `α : A' → A` is _j-orthogonal_ to the shape inclusion `ϕ ⊂ ψ`,
```

Comment:
What does "j" stand for here?

Reply:
Honestly, I have no idea!  @jonweinb?

Reply:
That's the name of the map the fibration is right orthogonal to. For example, covariant fibrations are the $i_0$-orthogonal fibrations where $i_0 : \{0\} \to \Delta^1$.

So $j$ is a parameter, not a constant part of the name.

Reply:
What is `j` in the definition of `is-j-orthogonal-to-shape`? The shape inclusion `ϕ ⊂ ψ`? Perhaps, this should be reflected in the documentation and in the name of the function (e.g. dropping `j` and leaving `is-orthogonal-to-shape`)?

Reply:
Yes,that's would it would be. And I agree with the suggestion.

Reply:
Then I will make those changes and also rename the file.